### PR TITLE
Bridge page

### PR DESCRIPTION
**Pull request by wildmolasses**

### CODE REVIEW

File: pages/bridge.tsx [hunk 1]
```diff
@@ -1,14 +1,143 @@
...
+import { useHasMounted } from "@/hooks/useHasMounted";
+import { useConfig } from "@/hooks/useConfig";
+import { useState } from "react";
+import { formatUnits } from "viem";
+import { classNames } from "@/util";
+import { ArrowLongRightIcon } from "@heroicons/react/20/solid";
+import { useBalances } from "@/hooks/useBalances";
+import { useFees } from "@/hooks/useFees";
+
+type BridgeDirection = "l1l2" | "l2l1";
```
[the remark this answers is not in the document]
Comment (wildmolasses) [reply pages/bridge.tsx:13]: agreed, enum works better! will make the change

File: config.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { avalancheFuji, polygonMumbai } from "wagmi/chains";
+
+export const chains = [avalancheFuji, polygonMumbai];
+
+type ConfigEntries = Record<
```
[the remark this answers is not in the document]
Comment (wildmolasses) [reply config.ts:5]: true, i'd be up for that

File: config.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  string,
+  {
+    chain: any;
+    wormholeChainId: number;
+    tokenAddress: `0x${string}`;
+    [key: string]: any;
```
[the remark this answers is not in the document]
Comment (wildmolasses) [reply config.ts:11]: sure, happy to remove the `any`. i'm considering converting this type to an interface. will ping with changes.

**Review by apbendi — COMMENTED** [review]
Looks good @wildmolasses, a good start for the bridge page! I have one question about what the env vars are supposed to be but otherwise I think it's good.

File: .env.template [hunk 4]
```diff
@@ -1 +1,2 @@
 NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=
+NEXT_PUBLIC_ALCHEMY_API_KEY=
```
[the remark this answers is not in the document]
Comment (wildmolasses) [reply .env.template:2]: this was a welcome addition from @alexkeating to get the rpc queries working on avalanche/mumbai. I wonder if we'll need this once we switch to goerli/OP goerli. Personally I used the first alchemy key in my project list and it seemed to work; perhaps alchemy key isn't tied to specific network after all, or else i just got lucky.

**Pull request merged**